test/backends/zfs.sh

```diff
@@ -0,0 +1,27 @@
+zfs_setup() {
+  local LXD_DIR
+  LXD_DIR=$1
+
+  if ! which zfs >/dev/null 2>&1; then
+    echo "couldn't find zfs binary"; false
+  fi
+
+  truncate -s 100G "${LXD_DIR}/zfspool"
+  # prefix lxdtest- here, as zfs pools must start with a letter, but tempdir
+  # won't necessarily generate one that does.
+  zpool create lxdtest-$(basename "${LXD_DIR}") "${LXD_DIR}/zfspool"
```

Comment:
Oh, sorry, missed the basename somehow, so I guess my concern is now the opposite, though it's still a reasonably large set of combinations that it may be okay. We'll move to uuid if we see collisions on Jenkins.